test/utils/init.js

```diff
@@ -20,6 +20,34 @@ chai.use((chaiAPI, utils) => {
       prettyDOM(document.activeElement),
     );
   });
+
+  chai.Assertion.addProperty('ariaHidden', function elementIsAccessible() {
+    const element = utils.flag(this, 'object');
+
+    // "An element is considered hidden if it, or any of its ancestors are not
```

Comment:
For integration test we could implement a custom matcher for `hidden` content as well. It won't catch everything (since it requires layout) but a crude implementation could just join `ariaHiden || !displayed || !visible`.